stateful-workload-filestore/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file and other cloudbuild.yaml files are used to ensure that
+# our public Docker images such as gcr.io/google-samples/hello-app:1.0
+# are rebuilt and updated upon changes to the repository.
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+    - 'build'
+    - '-t'
+    - 'us-docker.pkg.dev/google-samples/containers/gke/stateful-workload'
+    - '.'
+  dir: 'stateful-workload-filestore'
+
+# Push images.
+images:
+  - 'us-docker.pkg.dev/google-samples/containers/gke/stateful-workload'
```

Comment:
Praise: Thanks, @charlieyu1996, for figuring this out so pro-actively. The `cloudbuild.yaml` looks good!